cores/esp32/Stream.h

```diff
@@ -59,7 +59,10 @@ class Stream: public Print
 // parsing methods
 
     void setTimeout(unsigned long timeout);  // sets maximum milliseconds to wait for stream data, default is 1 second
-
+    unsigned long getTimeout(void)
```

Comment:
not in the header please :)

Reply:
addressed in d5c501f7687be64e5995189eb0be44c1fb53c7d8